akka-stream-tests/src/test/scala/akka/stream/impl/TimeoutsSpec.scala

```diff
@@ -130,6 +130,105 @@ class TimeoutsSpec extends AkkaSpec {
 
   }
 
+  "BackpressureTimeout" must {
+
+    "pass through elements unmodified" in assertAllStagesStopped {
+      Await.result(Source(1 to 100).backpressureTimeout(500.millis).grouped(200).runWith(Sink.head), 3.seconds) should === (1 to 100)
+    }
+
+    "succeed if backpressure notified" in assertAllStagesStopped {
+      val subscriber = TestSubscriber.probe[Int]()
+
+      Source(1 to 4)
+        .backpressureTimeout(500.millis)
+        .runWith(Sink.fromSubscriber(subscriber))
+
+      for (i ← 1 to 3) {
+        subscriber.requestNext(i)
+        subscriber.expectNoMsg(100.millis)
+      }
+
+      subscriber.requestNext(4)
+      subscriber.expectComplete()
+      subscriber.expectNoMsg(1.second)
```

Comment:
There is not. Removing it. I meant those delays in between sendNext - those verifies if idle stream or slow publisher won't make scheduler throw timeout exception by mistake.